src/taipy/core/_backup/_backup.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+
+import os
+import pathlib
+
+__BACKUP_FILE_PATH_ENVIRONMENT_VARIABLE_NAME = "TAIPY_BACKUP_FILE_PATH"
```

Comment:
Not sure it is a good idea, but you can do something like:
```
if preserve_file_path := os.getenv(__BACKUP_FILE_PATH_ENVIRONMENT_VARIABLE_NAME):
  def append_to_backup_file(new_file_path: str):
     ....
  def remove_from_backup_file(to_remove_file_path: str):
     ....
  def replace_in_backup_file(old_file_path: str, new_file_path: str):
     ...
else:
   def append_to_backup_file(*args, **kwargs):
     pass
  def remove_from_backup_file(*args, **kwargs):
     pass
  def replace_in_backup_file(*args, **kwargs):
     pass
```
Just because I don't like the 'if' in every function ^^' but it's a matter of taste ;)

Reply:
I'm all good!!

Reply:
Oh I did notice, if we do it this way, should we consider the case where `TAIPY_BACKUP_FILE_PATH` env variable is set during running of taipy? Because if we have the condition in placed like proposed, it will only check once and set for the entire life-time of the taipy project

Reply:
Very good point. Keep it as is :)